mindjunkies/lecture/signals.py

```
@@ -0,0 +1,25 @@
+# mindjunkies/lecture/signals.py
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from .models import LectureCompletion
+from mindjunkies.courses.models import Module
```

Ruff (0.8.2)

`mindjunkies.courses.models.Module` imported but unused

Remove unused import: `mindjunkies.courses.models.Module`

(F401)